### ISSUE TITLE
Authorizer AWS_IAM deploy fail

### ISSUE THREAD

**Issue by FbPalmabit** [serverless/serverless#3619]
## Description

I use `authorizer: aws_iam` as described in documentation (https://serverless.com/framework/docs/providers/aws/events/apigateway#http-endpoints-with-aws_iam-authorizers) but deploy fails with message: 
_Function "aws_iam" doesn't exist in this Service_

## Additional Data

* ***Serverless 1.13.1***:
* ***Node 6.10***:

**Comment by pmuens** [comment]
Hey @FbPalmabit thanks for reporting! 👍 

We've just merged the PR adding this authorizer, so Serverless `v1.13.1` does not include this feature yet (it's scheduled for `1.14`). However you can use the recent `master` to give it a spin!

**Comment by limandr** [comment]
According to previous comment from @pmuens, this was supposed to be fixed by 1.14. I'm running 1.48.4 and still seeing the same issue going on. 

```
Serverless: Packaging service...

Serverless: Excluding development dependencies...
 
  Serverless Error ---------------------------------------
 
  Function "iam_aws" doesn't exist in this Service
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              10.16.0
     Serverless Version:        1.48.4
     Enterprise Plugin Version: 1.3.3
     Platform SDK Version:      2.1.0
```
